roles/openshift_node/meta/main.yml

```diff
@@ -13,4 +13,5 @@ galaxy_info:
   - cloud
 dependencies:
 - { role: openshift_common }
+- { role: openshift_repos }
 - { role: docker }
```

Comment:
That said, it probably makes more sense to keep it than not, I do feel like an openshift_docker role to wrap the docker role may be warranted though.